### ISSUE TITLE
git up --version TypeError

### ISSUE THREAD

**Issue by xdbr** [msiemens/PyGitUp#34]
quick note: `git up --version` (at version 1.3.0) fails with a type error:

```
git up --version
GitUp version is: v1.3.0
Checking for updates...Traceback (most recent call last):
  File "/Users/dbr/.pyenv/versions/3.4.2/bin/git-up", line 9, in <module>
    load_entry_point('git-up==1.3.0', 'console_scripts', 'git-up')()
  File "/Users/dbr/.pyenv/versions/3.4.2/bin/gitup.py", line 579, in run
    GitUp(sparse=True).version_info()
  File "/Users/dbr/.pyenv/versions/3.4.2/bin/gitup.py", line 393, in version_info
    details = json.load(urlopen(PYPI_URL))
  File "/Users/dbr/.pyenv/versions/3.4.2/lib/python3.4/json/__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/Users/dbr/.pyenv/versions/3.4.2/lib/python3.4/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
```

**Comment by msiemens** [comment]
Thanks for reporting! Can you confirm that this problem is fixed on the latest `master`? (You can install via `pip install [--user] git+https://github.com/msiemens/PyGitUp/`)

**Comment by xdbr** [comment]
@msiemens: thank *you* for this module :)

can confirm it works, thanks!

**Comment by msiemens** [comment]
Great :) I've now released `v1.3.1` which includes the bugfix (6991262c6954bbed45d7cf1c29c53b7d7492a185)
